### PR TITLE
Add UnicaCity

### PR DESCRIPTION
**Pull request by Droby911**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress the images: https://compresspng.com/

**Comment by Droby911** [comment]
my bad

**Review by JakobBruening — APPROVED** [review]
Thank you!

File: minecraft_servers/unicacity/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "unicacity",
+  "nice_name": "UnicaCity",
+  "direct_ip": "server.unicacity.de",
+  "server_wildcards": [
+    "*.unicacity.de"
```
Comment (JakobBruening) [inline minecraft_servers/unicacity/manifest.json:6]: Ahh, there is still one thing.
This has to be `%.unicacity.de` :)

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/unicacity/manifest.json (hunk 1 above)
Comment (Droby911) [reply minecraft_servers/unicacity/manifest.json]: done

**Review by JakobBruening — APPROVED** [review]
Thank you! 👍🏼

**Pull request merged**